shared/constants/teams.tsx

```diff
@@ -928,7 +928,14 @@ export const dedupAddingMembeers = (
   const existing = [..._existing]
   for (const toAdd of toAdds) {
     if (!existing.find(m => m.assertion === toAdd.assertion)) {
-      existing.unshift(toAdd)
+      const role =
+        (toAdd.role === 'admin' || toAdd.role === 'owner') && toAdd.assertion.includes('@')
```

Comment:
:sheep: kinda bugged by a dedup function also doing this, I think it's fine to rely on the thing inserting these to not use an invalid role?

Reply:
Yeah you're right this isn't the right place for this.